src/ui_widgets/icon_view_tile.gd

```diff
@@ -0,0 +1,65 @@
+extends PanelContainer
+
+
+const NumberEdit = preload("res://src/ui_widgets/number_edit.gd")
+
+
+signal selected
+signal remove_requested
+signal texture_changed
+signal texture_size_changed
+
+
+@onready var select_button: Button = %SelectButton
+@onready var texture_rect: TextureRect = %TextureRect
+@onready var scale_label: Label = %ScaleLabel
+@onready var number_edit: NumberEdit = %NumberEdit
+@onready var remove_button: Button = %RemoveButton
+
+
+var texture: Texture2D
+
+
+var svg_markup: String:
+	set(value):
+		svg_markup = value
+		_update_texture()
+var texture_size: int:
+	set(value):
+		texture_size = value
+		if number_edit:
```

Comment:
Use `is_instance_valid(number_edit)` for explicitness. Also, apply earlier remarks for script formatting and spacing between setters.